sites/docs/docs/components/custom-echarts.md

```diff
@@ -66,7 +66,7 @@ from ${sales_by_country}
           },
           roam: false,
           nodeClick: false,
-          data: test_data,
+          data: data.test_data,
```

Comment:
I think this `data.` syntax is not necessary, and you reference the query directly as test_data

Reply:
I couldn't have the existing sample work on evidence 24.0.6 and had to do that change to have it working.

I've also noticed some weird behaviors that could be worth an issue:
- Dropdown doesn't appear to apply the filter on an `Echart` (while it works on a `BarChart`)
- New `queries` and related changes are not catched by `npm run dev`
- On Mac, if you browse the file, it creates the metadata file and then it throws a warning `ENOTDIR: not a directory, scandir 'static/data/bq/.DS_Store'`on `npm run sources`
- `evidence.settings.json` might not exist on the project creation resulting in warning errors too on command usage

Reply:
Ah somehow lost this comment - these are all really helpful thanks - I'm going to turn some of them into issues

Reply:
Great, about that PR, did manage to have it work with `test_data` or should we move forward with merging it?